src/main/java/org/sonarqube/A.java

```diff
@@ -1,4 +1,14 @@
 package org.sonarqube;
 
 public class A {
+  String message = "foo";
+
+  // TODO fix this
+  public String foo() {
+    message = null;
+    message.length();
```

Comment:
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_1@9
A "NullPointerException" could be thrown; "message" is nullable here.